keyboards/anne_pro/anne_pro.c

```diff
@@ -180,6 +180,13 @@ void keyboard_post_init_kb(void) {
 
 /* Start transmissions when the flag is set */
 void matrix_scan_kb(void) {
+    /* Check changes in Caps Lock, and set led */	
+	if (host_keyboard_leds() & (1<<USB_LED_CAPS_LOCK)){
```

Comment:
Maybe move this code from the general `matrix_scan_kb` to `anne_pro_lighting_update` as it seems more appropriate.

Reply:
Or it might be even better to use `led_set_kb`, which is only called on changes of the leds state.

Reply:
well, it has to be called in a polling way as it does not notify, but senses(if im correct), so it could be moved to `anne_pro_lighting_update` but definitely not to `led_set_kb`.

Reply:
moved to `anne_pro_lighting_update`